core/modules/views/plugins/views_plugin_style_grid.inc

```diff
@@ -12,17 +12,20 @@
  */
 class views_plugin_style_grid extends views_plugin_style {
   /**
-   * Set default options
+   * Set default options.
    */
   function option_definition() {
     $options = parent::option_definition();
-
     $options['columns'] = array('default' => '4');
```

Comment:
Why not change this to `$options['columns'] = array('default' => 4);` (numeric `4` instead of string), so that we can use `'#type' => 'number',` when defining the `$form['columns']` array?